addons/project/static/src/js/tours/project.js

```diff
@@ -66,67 +66,181 @@ registry.category("web_tour.tours").add('project_tour', {
+    content: markup(_t("You can also change the <b>state</b> of the task here.")),
+    position: "bottom",
+    run: "click",
+}, {
+    trigger: ".o_field_project_task_state_selection .dropdown-toggle",
+    content: markup(_t("Click again to close the dropdown.")),
+    position: "left",
+    run: "click",
+}, {
+    trigger: 'a[name="sub_tasks_page"]',
+    content: _t('Open sub-tasks notebook section.'),
+    run: 'click',
+}, {
+    trigger: '.o_field_subtasks_one2many .o_list_renderer a[role="button"]',
+    content: _t('Add a sub-task.'),
+    run: 'click',
+}, {
+    trigger: '.o_field_subtasks_one2many div[name="name"] input',
+    content: markup(_t('Give the sub-task a <b>name</b>.')),
+    run: 'text New Sub-task'
+}, {
+    trigger: ".breadcrumb-item:not(.active):last",
+    extra_trigger: '.o_form_project_tasks',
+    content: markup(_t("Let's go back to the <b>kanban view</b> to have an overview of your next tasks.")),
+    position: "right",
+    mobile: false,
+    run: 'click',
+}, {
+    trigger: ".o_back_button",
+    extra_trigger: '.o_form_project_tasks',
+    content: markup(_t("Let's go back to the <b>kanban view</b> to have an overview of your next tasks.")),
+    position: "right",
+    mobile: true,
+    run: 'click',
+}, {
+    trigger: '.o_kanban_record .o_field_project_task_state_selection button',
+    content: markup(_t("Check the <b>state</b> of the task here again.")),
+    position: "bottom",
+}, {
+    trigger: '.o_kanban_record .o_field_project_task_state_selection .dropdown-menu .dropdown-item:has(.fa-times-circle)',
+    content: markup(_t("You can use the last two <b>states</b> proposed <i>Canceled</i> and <i>Done</i> to <b>close</b> the task, \
+    closed task won't appear in the kanban view by default, try to set this task as <i>Canceled</i>.")),
+    position: "right",
+}, {
+    trigger: '.o_kanban_record:has(.o_field_project_task_state_selection .dropdown-toggle:has(.fa-times-circle))',
+    content: _t("You can see that the Task was blurred but it won't disapear until you reload this page, click on the task."),
+    position: "right",
 }, {
     trigger: ".breadcrumb-item:not(.active):last",
     extra_trigger: '.o_form_project_tasks',
+    content: markup(_t("Go back to the <b>kanban view</b>.")),
+    position: "right",
+    mobile: false,
+    run: 'click',
+}, {
+    trigger: ".o_back_button",
+    extra_trigger: '.o_form_project_tasks',
     content: markup(_t("Let's go back to the <b>kanban view</b> to have an overview of your next tasks.")),
     position: "right",
+    mobile: true,
     run: 'click',
+}, {
+    trigger: ".o_control_panel_navigation button:has(.fa-search)",
+    content: _t("Open the filter search input."),
+    position: "bottom",
+    mobile: true,
+    run: 'click',
+}, {
+    trigger: ".o_control_panel_actions .o_searchview_input_container .o_searchview_facet .o_facet_remove",
+    content: _t("Remove the default open task filter."),
```

Comment:
```suggestion
    content: _t("Remove the default <i>Open Tasks</i> filter."),
```
or something else to make it stand out from the rest of the sentence